Gaming The Syllabus/Assets/Scripts/Riddles/SyllabusRiddleManager.cs

```diff
@@ -50,8 +50,13 @@ private static void LoadRiddlesFromXML(string filePathToRiddlesFolder)
         {
             // We end up with the Assets folder in the path twice since our XmlUtilities class uses the app data path which
             // is the path to the Assets folder, so we just remove it here.
-            riddles.Add(XmlUtilities.Deserialize<Riddle>(riddlePath.Replace("Assets/", "")));
+            Riddle riddle = XmlUtilities.DeserializeFromAbsolutePath<Riddle>(riddlePath);
+            riddles.Add(riddle);
         }
+
+        Debug.Log(riddles.Count);
+
+        SyllabusRiddleUIManager.DisplayRiddle(riddles[0]);
```

Comment:
I know this is for testing purposes only, but we definitely need to make sure that the UI for riddles isn't automatically displayed on start in the final build.

Reply:
Good catch, this was changed in the latest PR for this sprint!